knowledge-base/codacy/codacy-analysis-cli-action/action-security.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Codacy Analysis CLI'
+outbound-endpoints:
+  - fqdn: raw.githubusercontent.com
+    port: 443
+    reason: to fetch source code
+  - fqdn: '*.docker.io'
+    port: 443
+    reason: to access docker 
+  - fqdn: auth.docker.io
```

Comment:
`auth.docker.io` will get covered under `*.docker.io`, so you can remove `auth.docker.io`